pkg/security/ldapclient/client.go

```diff
@@ -77,7 +118,7 @@ func (l *ldapClientConfig) Connect() (ldap.Client, error) {
 
 	switch l.scheme {
 	case ldaputil.SchemeLDAP:
-		con, err := ldap.Dial("tcp", l.host)
+		con, err := ldapDial("tcp", l.host)
```

Comment:
We should synchronize with @ardaguclu on his https://github.com/openshift/library-go/pull/1378 and use https://github.com/go-ldap/ldap/blob/master/conn.go#L120 instead of replacing the package's dial.

Reply:
Looked again and this isn't actually possible, because `DialWithDialer` takes  a concrete`*net.DIaler` and not an interface, so we can't pass the proxy dialer there. Also, it is only used in `DialURL`, which we don't use